src/VisualStudio/Core/Next/Progression/VisualBasicGraphProvider.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.GraphModel;
+using Microsoft.VisualStudio.Language.Intellisense;
+using Microsoft.VisualStudio.LanguageServices.Implementation.Progression;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.LanguageServices.CSharp.Progression
+{
+    [GraphProvider(Name = "VisualBasicRoslynProvider", ProjectCapability = "VisualBasic")]
+    internal sealed class VisualBasicGraphProvider : AbstractGraphProvider
+    {
+        [ImportingConstructor]
+        public VisualBasicGraphProvider(
```

Comment:
Given that this is identical to the C# one, is it possible to have a single provider instead that is tagged with both Capabilities?  Doing that would make it more understandable why these are in the language neutral assembly.

If not, can you add a comment explaining why they are where they are?